File: LorePdks.WEB/src/app/core/models/RolDto.ts
import { baseDto } from './baseDto';
import { EkranDto } from './EkranDto';

/**
 * Rol bilgilerini temsil eden model
 */
export class RolDto extends baseDto {
  rolAdi: string;
  aciklama: string;
  ekranlar: EkranDto[];
}

File: LorePdks.COMMON/DTO/Yetki/Rol/RolDTO.cs
using LorePdks.COMMON.DTO.Base;
using LorePdks.COMMON.DTO.Yetki.Ekran;
using System;
using System.Collections.Generic;

namespace LorePdks.COMMON.DTO.Yetki.Rol
{
    public class RolDTO : BaseDTO
    {
        public string rolAdi { get; set; }
        public string aciklama { get; set; }
        public List<EkranDTO> ekranlar { get; set; }
    }
}